test/suites/nodeclaim/garbage_collection_test.go

```diff
@@ -147,3 +150,44 @@ var _ = Describe("NodeClaimGarbageCollection", func() {
 		env.EventuallyExpectNotFound(node)
 	})
 })
+
+func ExpectInstanceProfileCreated(instanceProfileName *string) {
+	By("creating an instance profile")
+	createInstanceProfile := &iam.CreateInstanceProfileInput{
+		InstanceProfileName: instanceProfileName,
+		Tags: []*iam.Tag{
```

Comment:
Is there a common set of tags that we should add to all of our resources outside of our common tags?

Reply:
What tags were you thinking? All we need is the one tag to discover for cleanup, since this is only being used for the one-off.

Reply:
I was mainly curious if we had any other tags to at we were adding. If not, then this is fine with me